stable/kommander/requirements.lock

```diff
@@ -4,10 +4,10 @@ dependencies:
   version: 0.1.4
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: v0.4.2
+  version: 0.4.2
```

Comment:
does anyone know how that `v` keeps being added? `helm dep update` reoves it for me 🤷‍♂